index.css

```diff
@@ -122,7 +137,7 @@ h2 {
 
 
 /***DESKTOP***/
-@media screen and (min-width: 1200px) {
+@media screen and (min-width: 1250px) {
```

Comment:
This is gonna increase the size the screen has to be to be in desktop mode. Is that what you want?

Reply:
Yes, just by 50px so it doesn't clash with the .actions bubble. I tested on two screens, but let me know if it's throwing any issues for your screens!